app/components/dashboard/backed-chats.tsx

```diff
@@ -59,33 +61,19 @@ export default function BackedChats() {
         </div>
       ) : (
         <div className="flex flex-col">
-          {loadingDialogs && (
-            <div className="text-center">
-              <Loading text={'Loading chats...'} />
+          {Object.values(dialogIdMap).map(({ dialogInfo, latestBackup }) => (
```

Comment:
I am curious if we are maintaining sort order here of newest backup first, given that it's an object key map.

Reply:
to me it looks like it’s keeping things in alphabetical order. I can open another PR to make it follow the backup order instead